server/network.js

```diff
@@ -347,89 +331,90 @@ network.readNetwork_ = function(networkFile) {
   var isFirst = true;
   var valueNames = [];
   var lines = fs.readFileSync(networkFile).toString().split('\n');
-  for (var lineNum in lines) {
-    var line = lines[lineNum];
-    if (!validFile) continue;
+  lines.forEach(function(line) {
+    if (!validFile) return;
     var parts = line.split(/[\t\s]+/);
+    var source = parts[0];
+    var target = parts[1];
     if (parts.length < 3) {
       validFile = false;
-      continue;
+      return;
```

Comment:
@return {Object} is too vague. Use more explicit types or typedefed return types. Return null on error.